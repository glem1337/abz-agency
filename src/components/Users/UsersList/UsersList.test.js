import React from 'react';
import {shallow} from 'enzyme'
import UsersList from "./UsersList";


describe('Users List', () => {

    const props = {
        usersList: [
            {id: 0},
            {id: 1},
        ],
    };

    const usersList = shallow(<UsersList {...props} />)

    it('renders properly', () => {
        expect(usersList).toMatchSnapshot();
    });

    it('should be 2 users', () => {
        expect(usersList.find('User')).toHaveLength(2);
    });


});