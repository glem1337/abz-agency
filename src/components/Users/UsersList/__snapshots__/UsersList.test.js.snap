// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Users List renders properly 1`] = `ShallowWrapper {}`;
